Tools/scripts/deepfreeze.py

```diff
@@ -114,6 +114,13 @@ def __init__(self, file: TextIO):
         self.write('#include "internal/pycore_gc.h"')
         self.write('#include "internal/pycore_code.h"')
         self.write("")
+        self.write("#define INTERN_STRING(name)                  \\")
+        self.write("do                                           \\")
+        self.write("{                                            \\")
+        self.write("    PyObject *str = (PyObject *)name;        \\")
+        self.write("    PyUnicode_InternInPlace(&str);           \\")
```

Comment:
This adds the string object to the interning dict, but references to the string elsewhere (e.g. in co_names or co_consts) won't be affected.

Reply:
I have reworked this with the already existing code to intern strings in codeobject.c and also resoved the conflicts with other deepfreeze changes.